README.md

```diff
@@ -228,7 +270,8 @@ When it launches it will send PR to update all the repos selected in step (2.2).
     # Location of a `.scala-steward.conf` file with default
     # values.
     # 
-    # If the provided file is missing the action will fail.
+    # If you specify a file and it does not exist in the 
```

Comment:
This needs to be changed in [`action.yml`](https://github.com/scala-steward-org/scala-steward-action/blob/master/action.yml) as it is generated from there.

Reply:
Added.